src/Microsoft.Health.Operations.Functions.UnitTests/Management/PurgeOrchestrationInstanceHistoryTests.cs

```diff
@@ -24,16 +27,16 @@ public class PurgeOrchestrationInstanceHistoryTests
     private readonly DateTime _utcNow;
     private readonly TimerInfo _timer;
     private readonly PurgeHistoryOptions _purgeConfig;
-    private readonly PurgeOrchestrationInstanceHistory _purgeTask;
     private readonly IDurableOrchestrationClient _durableClient;
+    private PurgeOrchestrationInstanceHistory _purgeTask;
```

Comment:
This can still be `readonly`, right? You just need to modify the values of the `_purgeConfig` before testing

Reply:
Done